tests/phpunit/tests/rest-api/rest-global-styles-controller.php

```diff
@@ -513,4 +513,23 @@ public function test_get_theme_items() {
 		);
 		$this->assertSameSetsWithIndex( $data, $expected );
 	}
+
+	/**
+	 * @covers WP_REST_Global_Styles_Controller::get_available_actions
+	 */
+	public function test_assign_edit_css_action_admin() {
+		wp_set_current_user( self::$admin_id );
+
+		$request = new WP_REST_Request( 'GET', '/wp/v2/global-styles/' . self::$global_styles_id );
+		$request->set_param( 'context', 'edit' );
+		$response = rest_do_request( $request );
+		$links    = $response->get_links();
+
+		// Admins can only edit css on single site.
+		if ( is_multisite() ) {
+			$this->assertArrayNotHasKey( 'https://api.w.org/action-edit-css', $links );
+		} else {
+			$this->assertArrayHasKey( 'https://api.w.org/action-edit-css', $links );
+		}
+	}
```

Comment:
I used the post-controller tests as a blueprint here https://github.com/WordPress/wordpress-develop/blob/6.1/tests/phpunit/tests/rest-api/rest-posts-controller.php#L4903-L4918.

Reply:
Good job @Mamaduka 👏